Angular/Assignment-1/src/alert-button/alert-button.component.html

```diff
@@ -0,0 +1,11 @@
+<html>
+<head>
+	<title>Button</title>
+	<script src="button.js"></script>
```

Comment:
no need to add html and body, we can directly use the tags

Reply:
ok